Core/LocalAdmin.cs

```diff
@@ -181,7 +181,17 @@ internal async Task Start(string[] args)
         {
             await LoadJsonOrTerminate();
 
-            if (DataJson!.EulaAccepted == null)
+            var isEulaNotAccepted = DataJson!.EulaAccepted == null;
+
+            var autoEula = Environment.GetEnvironmentVariable("SLLA_EULA")?.ToUpperInvariant() is "1" or "TRUE"
+                           || args.Contains("--eula");
+
+            if (isEulaNotAccepted && autoEula)
+            {
+                ConsoleUtil.WriteLine("EULA was auto-accepted due to the provided argument or environment variable", ConsoleColor.Yellow);
+                DataJson.EulaAccepted = DateTime.UtcNow;
+            }
+            else if (isEulaNotAccepted)
```

Comment:
Both if and else if statements check for the same variable. Just make a one big if for `if (DataJson!.EulaAccepted == null)` (currently defined as `isEulaNotAccepted`) and inside the big if put one if and one else statements.